src/main/java/io/github/genomicdatainfrastructure/daam/api/ApplicationNotFoundExceptionMapper.java

```diff
@@ -21,7 +22,9 @@ public Response toResponse(ApplicationNotFoundException exception) {
         var errorResponse = new ErrorResponse(
                 "Application Not Found",
                 NOT_FOUND.getStatusCode(),
-                exception.getMessage()
+                exception.getMessage(),
+                Collections.singletonList(
```

Comment:
important: Please pass an empty collection when there is no error message. You can also use `List.of()` instead of `Collections.singletonList`.